lib/Traits/Decorators/anyTrait.ts

```diff
@@ -19,6 +20,8 @@ export default function anyTrait(options: TraitOptions) {
 }
 
 export class AnyTrait extends Trait {
+  readonly decoratorForFlattened = computed.struct;
+
```

Comment:
Cesium3DTilesCatalogItem uses anyTrait for specifying styles. Using structural comparison here will prevent unnecessary recomputes of `mapItems` when style is object changes without actual values changing (eg when stratum is recreated during story scene change)